packages/components/fields/number-field/README.md

```diff
@@ -67,7 +67,7 @@ export default Example;
 | `placeholder`          | `string`                                                             |          |           | Placeholder text for the input                                                                                                                                                                                                                                        |
 | `min`                  | `number`                                                             |          |           | Value is used as `min` property on input field                                                                                                                                                                                                                        |
 | `max`                  | `number`                                                             |          |           | Value is used as `max` property on input field                                                                                                                                                                                                                        |
-| `step`                 | `number`                                                             |          |           | Value is used as `step` property on input field                                                                                                                                                                                                                       |
+| `step`                 | `<number\|enum>`                                                     |          |           | Value is used as `step` property on input field                                                                                                                                                                                                                       |
```

Comment:
Is there a way to indicate that we support numbers and "any"?

Reply:
Not currently. It's automated by the `generate-readme` script.
But you could add a suggestion as a second line to the description of the prop stating something like "Use the value `any` for inputs which accept an unpredictable amount of decimals."